minerl/Malmo/Minecraft/src/main/java/com/microsoft/Malmo/MissionHandlers/NearbySmeltCommandsImplementation.java

```diff
@@ -19,70 +19,40 @@
 
 package com.microsoft.Malmo.MissionHandlers;
 
-import io.netty.buffer.ByteBuf;
+import com.microsoft.Malmo.Utils.MineRLTypeHelper;
```

Comment:
TODO: remove modifications to this file because I don't have the bandwidth to test these changes

Reply:
(changes to smelt)